src/extractors/generic/next-page-url/extractor.test.js

```diff
@@ -7,12 +7,12 @@ const fs = require('fs');
 
 describe('GenericNextPageUrlExtractor', () => {
   it('returns most likely next page url', () => {
-    const html = fs.readFileSync('./fixtures/ars.html', 'utf8');
+    const html = fs.readFileSync('./fixtures/arstechnica.com.html', 'utf8');
     const $ = cheerio.load(html);
     const url =
-      'http://arstechnica.com/gadgets/2016/08/the-connected-renter-how-to-make-your-apartment-smarter/';
+      'https://arstechnica.com/gadgets/2016/08/the-connected-renter-how-to-make-your-apartment-smarter/';
```

Comment:
test fails with the old `http://arstechnica.com/gadgets/2016/08/the-connected-renter-how-to-make-your-apartment-smarter/` and passes with newer `https://arstechnica.com/gadgets/2016/08/the-connected-renter-how-to-make-your-apartment-smarter/`